pagopa/pn-frontend-e2e
# PN-9432-e-2-e-destinatario-sezione-stato-piattaforma-disservizi-in-corso

## Short description
There isn't a test related to the **PN-9342** as you can see in the [confluence page](https://pagopa.atlassian.net/wiki/spaces/PN/pages/868811083)

## List of changes proposed in this pull request
- added new feature file `052_A1_visualizzazioneDisserviziIncorsoPF`
- renamed some folders in order to have the same name with every APP

## How to test
Simply run `mvn clean test -Dcucumber.filter.tags=@TA_PF_VisualizzaDisservizioInCorso` and verify if the test works fine.
